static/app/utils/replays/timer.tsx

```diff
@@ -33,7 +34,7 @@ export class Timer extends EventTarget {
    */
   start(seconds?: number) {
     this._pausedAt = 0;
-    this._start = window.performance.now() - (seconds ?? 0);
+    this._start = window.performance.now() - (seconds ?? 0) / this._speed;
```

Comment:
I don't understand why this is here. It seems like `start(seconds)` should just set the time directly. I don't follow why we'd adjust that requested timestamp, like if the video is paused we'd still want to start at `seconds`. I'm probably just missing something about the math, or about what `_start` does i guess.

Reply:
i'll add a comment explaining the math!

Reply:
```
// we're dividing by the speed here because we mutiply
    // by the same factor up in step(). doing the math,
    // you'll see that this._time turns out to be just `seconds`.
```

awesome, that's what I needed 👍